game/game.py

```diff
@@ -415,46 +423,51 @@ def initialize_turn(self) -> None:
             self.compute_transit_networks()
         self.ground_planners = {}
 
-        self.blue_procurement_requests.clear()
+        if for_player:
+            self.blue_procurement_requests.clear()
         self.red_procurement_requests.clear()
 
         with logged_duration("Procurement of airlift assets"):
             self.transfers.order_airlift_assets()
         with logged_duration("Transport planning"):
             self.transfers.plan_transports()
 
-        with logged_duration("Blue mission planning"):
-            if self.settings.auto_ato_behavior is not AutoAtoBehavior.Disabled:
-                blue_planner = CoalitionMissionPlanner(self, is_player=True)
-                blue_planner.plan_missions()
-
-        with logged_duration("Red mission planning"):
-            red_planner = CoalitionMissionPlanner(self, is_player=False)
-            red_planner.plan_missions()
+        for coalition in ["Red", "Blue"]:
+            is_player = coalition == "Blue"
+            with logged_duration(f"{coalition} mission planning"):
+                if not is_player or (
+                    is_player
+                    and for_player
+                    and self.settings.auto_ato_behavior is not AutoAtoBehavior.Disabled
+                ):
+                    mission_planner = CoalitionMissionPlanner(self, is_player=is_player)
+                    mission_planner.plan_missions()
 
         for cp in self.theater.controlpoints:
             if cp.has_frontline:
                 gplanner = GroundPlanner(cp, self)
                 gplanner.plan_groundwar()
                 self.ground_planners[cp.id] = gplanner
 
-        self.plan_procurement()
+        if not is_reinit:
+            self.plan_procurement(for_player)
 
-    def plan_procurement(self) -> None:
+    def plan_procurement(self, for_player: bool) -> None:
```

Comment:
(same as above, `opfor_only` is clearer)

Reply:
i changed the whole thing to `for_coalition: List[str] = ["Blue","Red"]` so that you can set for which coalition you want it, or both